ceph-build-os/config/definitions/ceph-build-os.yml

```diff
@@ -0,0 +1,62 @@
+- job:
+    name: ceph-build-os
+    project-type: freestyle
+    defaults: global
+    concurrent: true
+    node: huge && (ceph_build_xenial) && rebootable
+    display-name: 'ceph: Build OS'
+    quiet-period: 5
+    block-downstream: false
+    block-upstream: false
+    retry-count: 3
+    properties:
+      - github:
+          url: https://github.com/ceph/ceph/
+    logrotate:
+      daysToKeep: 15
+      numToKeep: 300
+      artifactDaysToKeep: -1
+      artifactNumToKeep: -1
+
+    parameters:
+        - string:
+            name: DIST
+            description: "The distribution codename like xenial,centos,jessie"
+
+        - string:
+            name: DVER
+            description: "The distribution codename like C7.2, U16.04, D8"
+
+        - string:
+            name: VERSION
+            description: "The release of that particular build : 1.0.0, 1.1.0, ..."
+
+        - string:
+            name: REPOSITORY
+            description: "URL of the package repository, like http://debian.mirrors.ovh.net/debian/"
```

Comment:
I am guessing that this is something that changes from distro to distro. Why do we need to pass in the full URL for a repository? Can't the job figure out which one to use and default to that unless we want one explicitly? 

And what would the URL be for an RPM distro?